controllers/order.js

```diff
@@ -0,0 +1,139 @@
+//const Order = require("../models/Order");
+const Order = require("../models/Order");
+const { validationResult } = require("express-validator");
+
+
+exports.createOrder = async (req, res, next) => {
+  try {
+    const {
+      user,
+      items,

```

Comment:
what type of data are you expecting here?

can you send example here

Reply:
objects ids